Source/ASTextNode.mm

```diff
@@ -1010,7 +1011,8 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event
   NSUInteger lastCharIndex = NSIntegerMax;
   BOOL linkCrossesVisibleRange = (lastCharIndex > range.location) && (lastCharIndex < NSMaxRange(range) - 1);
 
-  if (range.length > 0 && !linkCrossesVisibleRange && linkAttributeValue != nil && linkAttributeName != nil) {
+  if (_alwaysHandleTruncationTokenTap ||
```

Comment:
Sorry I wasn't very clear. What I meant in my previous comment is that we can check this property early (after the `_passthroughNonlinkTouches` check) and return immediately. We wouldn't bother calling `_linkAttributeValueAtPoint:` because its results wouldn't be considered anyways.

Reply:
Also, you also need to have the lock before accessing the ivar. One simple way for now is using the getter.

Reply:
@nguyenhuy Thanks for your suggestion.